mmv1/openapi_generate/openapi/developerconnect_public_openapi3_0_v1.json

```diff
@@ -0,0 +1,5264 @@
+{
```

Comment:
I think we can delete this? We use them during generation but don't typically commit them.